distro/docker/Dockerfile

```diff
@@ -26,39 +26,57 @@ RUN apt-get update
 
 # Install packages required to build and test
 RUN apt-get install -y \
-  wget \
-  git \
   build-essential \
   cmake \
   doxygen \
+  git \
   graphviz \
-  libglib2.0-dev \
   libeigen3-dev \
-  libqt4-dev \
-  libvtk5-dev \
-  libvtk5-qt4-dev \
-  libvtk-java \
+  libexpat-dev \
+  libfreetype6-dev \
+  libglib2.0-dev \
+  libjpeg-dev \
+  libqt5webkit5-dev \
+  libqt5x11extras5-dev \
+  libtiff5-dev \
+  libxml2-dev \
+  libxt-dev \
   python-coverage \
   python-dev \
   python-lxml \
   python-numpy \
   python-pip \
   python-sphinx \
-  python-vtk \
   python-yaml \
+  python2.7-dev \
+  qt5-default \
+  qtbase5-dev \
+  qtbase5-private-dev \
+  qtmultimedia5-dev \
+  qttools5-dev \
+  wget \
   xvfb
 
 RUN pip install sphinx-rtd-theme
 
+RUN wget https://d2mbb5ninhlpdu.cloudfront.net/vtk/vtk-v7.1.1-1584-g28deb56-qt-5.5.1-xenial-x86_64.tar.gz
+RUN tar -xvzf vtk-v7.1.1-1584-g28deb56-qt-5.5.1-xenial-x86_64.tar.gz
```

Comment:
This is more of a package management issue versus a build system.
If you think it makes it easier for users, I can add it to the superbuild, as an option that supersedes `USE_SYSTEM_VTK`. Something like, `INSTALL_VTK_PACKAGE`. Thoughts?